ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/restore/viewmodel/RestoreViewModel.kt

```diff
@@ -54,7 +55,7 @@ class RestoreViewModel(application: Application, savedStateHandle: SavedStateHan
         // viewModelScope is constructed with Dispatchers.Main.immediate, so this will
         // update the save state as soon as a change occurs.
         userWordList.current.collectWith(viewModelScope) {
-            savedStateHandle.set(KEY_WORD_LIST, it)
+            savedStateHandle[KEY_WORD_LIST] = ArrayList(it)
```

Comment:
It has to be mutable list here, right? What about a bit more general MutableList?

Reply:
ArrayList is Serializable, which meets the requirements of saved instance state needing to be Serializable or Parcelable.